jenkinsci/workflow-cps-plugin
# Upgrade ACE Editor from 1.2.2 to 1.15.0

Upgrade [ACE Editor](https://ace.c9.io/) from 1.2.2 (released on October 28, 2015) to 1.15.0 (released on January 25, 2023).

### Implementation

Followed the example in the [Webpack demo here](https://github.com/ajaxorg/ace/blob/159aa70d551530f2866fb0006fad37bd75e60dda/demo/webpack/demo.js).

### Testing done

- Create a new Pipeline job and try samples
- Autocomplete Pipeline-specific snippets
- Introduce a syntax error and verify validation fails
- Fix a syntax error and verify validation succeeds
- Resize the editor
